### ISSUE TITLE
w3m clear image script clears to the terminal background 

### ISSUE THREAD

**Issue by Danielgb23** [vifm/vifm#736]
I use the w3mimagedisplay binary to display the image previews which works fine with:

```
fileviewer *.bmp,*.jpg,*.jpeg,*.png,*.gif,*.xpm
         \ ~/.scripts/w3mimgdisplay.sh %px %py %pw %ph %c
	 \ %pc
         \ ~/.scripts/w3mimgclear.sh %px %py %pw %ph
```
But when it clears it eats up the vifm gray background showing up a hole into my terminals transparent background like this:

![image](https://user-images.githubusercontent.com/33136385/140441021-29de1f3b-9dc9-405e-ac1a-1b583eb06ba5.png)

Clear script:
```
#!/bin/bash

W3MIMGDISPLAY='/usr/lib/w3m/w3mimgdisplay'
FONTH=18 # Size of one terminal row
FONTW=10 # Size of one terminal column

X=$1
Y=$2
COLUMNS=$3
LINES=$4

x=$((FONTW * X))
y=$((FONTH * Y))

erase="6;$x;$y;$(( FONTW*COLUMNS ));$(( FONTH*LINES ))\n3;"
echo -e "$erase" | $W3MIMGDISPLAY
```

**Comment by xaizek** [comment]
I don't have such problem, maybe transparency is the issue. I found that `w3mimgdisplay` has `-bg` option. You can try specifying it (e.g., `-bg red`).

**Comment by Danielgb23** [comment]
> I don't have such problem, maybe transparency is the issue. I found that `w3mimgdisplay` has `-bg` option. You can try specifying it (e.g., `-bg red`).

Setting a background on both the imagedisplay and the clear just leaves the background wherever the image is transparent. But it still leaves the transparent artifacts on the rest of the vifm pannel. 

I think the solution would be doing a clear where we display an entirely transparent image with the background color desired and with the right size to cover all of the part that can be used by the images.

**Comment by xaizek** [comment]
> Setting a background on both the imagedisplay and the clear just leaves the background wherever the image is transparent. But it still leaves the transparent artifacts on the rest of the vifm pannel.

Doesn't that fix the issue if you use background color that matches pane color?

If `w3mimgdisplay` can't do it, Vifm can't help as it's not the one doing drawing of the image.

**Comment by Danielgb23** [comment]
Yes. I just need to execute this instead of the clear command

**Comment by xaizek** [comment]
So doing `echo -e "$erase" | $W3MIMGDISPLAY -bg 'gray'` works?

**Comment by Danielgb23** [comment]
> So doing `echo -e "$erase" | $W3MIMGDISPLAY -bg 'gray'` works?

Nope. I have to figure out how to print a transparent image using the background option

**Comment by xaizek** [comment]
Have you tried ARGB format (`#ff000000`)? `-bg` doesn't seem to actually change erasure color for me (uses terminal background). Maybe consider using [vifmimg](https://github.com/cirala/vifmimg), I think Ueberzug shouldn't have this issue with colors.
